test/dummy/config/storage.yml

```diff
@@ -0,0 +1,34 @@
+test:
```

Comment:
fyi - placeholder file needed for rails 7 onwards.

Reply:
❓  Since `large_text_field` doesn't require the storage engine for testing/functionality, could we not include it in the dummy application so we don't have to continuously add additional configuration?